cloud/wfm/src/index.ts

```diff
@@ -0,0 +1,33 @@
+// Interfaces
+import Executor from './executor/Executor';
+import ProcessInstance from './process-instance/ProcessInstance';
+import Process from './process/Process';
+import Result from './result/Result';
+import Task from './task/Task';
+
+// Implementations
+import ExecutorImpl from './executor/Executor';
```

Comment:
[Optional] `ExecutorImpl` - BaseExecutor or DefaultExecutor or ProcessExecutor - remember we were saying to avoid Impl or Interface.